src/_playground/App.scss

```diff
@@ -96,12 +95,28 @@ $library-color: #61dafb;
             }
         }
 
+        @media (max-width: $mediaMaxWidth) {
+            .frDocs-Sidebar.sidebar-hidden{
+                        margin-left: -20rem;
+                        overflow-x: hidden;
+            }
+
+            code {
+                display: flex;
+                flex-wrap: wrap; //remove x-axis scrolling 
+            }
+        }
+
         .frDocs-Content {
             flex-grow: 1;
             overflow-y: scroll;
             background-color: white;
             padding: 2rem;
 
+            .sidebar-toggle{
+                visibility: hidden;
```

Comment:
I think this should toggle between `display: none` and `display: block` (not `visibility`).